qiskit/circuit/quantumcircuit.py

```diff
@@ -835,7 +835,7 @@ def compose(
                 this can be anything that :obj:`.append` will accept.
             qubits (list[Qubit|int]): qubits of self to compose onto.
             clbits (list[Clbit|int]): clbits of self to compose onto.
-            front (bool): If True, front composition will be performed (not implemented yet).
+            front (bool): If True, front composition will be performed.  This is not possible within
```

Comment:
I think this might be missing something? Maybe within classical control flow block?

Reply:
oh yeah, whoops!  I must have got distracted while writing it.  Fixed in 4b53b55a1.